### DISCUSSION TITLE
[BUG] GitHub App installations

### DISCUSSION THREAD

**Discussion by nehzata** [community/community#37996] [Apps, API and Webhooks]
Hi,

I think there's a **serious bug with App tokens**. I have come across a case where `https://api.github.com/user/installations` is returning an installation that I've confirmed has never happened and is impossible. The token is getting access to repositories it shouldn't! Can someone contact me asap please.

Ali,

**Comment by nehzata** [comment]
@ossanna16 are you able to help with this please?

**Comment by ghostinhershell** [comment]
Hey @nehzata, thanks so much for reaching out.

The best next course of action here would be to [submit a bug report](https://support.github.com/contact/bug-report) so our Support team can investigate.

**Reply by choufleur123** [reply]
Salut mec

**Reply by choufleur123** [reply]
Je suis nouveau sur le groupe j'aimerais tu sois mon mentor

**Reply by ltkiet39475** [reply]
TXXo4Ugkf1EpATNVxNKDXq6qwbQ6LX1UWT

**Comment by nehzata** [comment]
Thanks @ghostinhershell !

**Reply by ghostinhershell** [reply]
No problem!

**Reply by Ngonj** [reply]
Ok ok

**Comment by sharifboydev** [comment]
Thanks
